error-handling/readme.md

```diff
@@ -0,0 +1,83 @@
+# Error Handling
+
+## Learning Objectives
+
+```objectives
+- [ ] Give examples of good and bad error messages
+- [ ] Identify appropriate points in code to check for errors
+```
+
+## Understanding What Went Wrong
+
+We have already seen examples of developer questions which can help us fix problems in our code:
+
+1. What I _did_
+2. What I _expected_
+3. What actually _happened_
+
+The problem is that figuring out what happened can be really tricky if the developer doesn't do a good job of **error handling**. In this workshop we're going to discuss error messages, why they are important and what makes some more helpful than others.
+
+What makes an error message "good" depends on who it is intended for. For example, the runtime errors you may have seen while working through the exercises are great for developers while they're working on the code but they aren't very useful for the end user looking at an app on their phone. Knowing the most appropriate way to communicate problems is a key skill for a developer.
+
+### Setup
+
+- **Trainees**: Go to the `error-handling` folder in the [Workshops repo](https://github.com/CodeYourFuture/CYF-Workshops/), navigate to the `webpage` folder and open `index.html` in a browser.
```

Comment:
We can either have them clone and run this locally, or we can point them at a hosted version of the frontend, or both. Do you have a strong preference?